subsystem/wrist.py

```diff
@@ -0,0 +1,83 @@
+from toolkit.subsystem import Subsystem
+from toolkit.motors.ctre_motors import TalonFX
+from phoenix6.hardware import cancoder
+
+import math
+from math import pi
+from units.SI import radians
+from toolkit.utils.toolkit_math import bounded_angle_diff
+
+import config
+import constants
+
+class Wrist(Subsystem):
+    def __init__(self):
+        super().__init__()
+        self.feed_motor: TalonFX = TalonFX(
+            config.wrist_feed_id,
+            config.foc_active,
+            inverted = False,
+            config = config.WRIST_FEED_CONFIG
+        )
+        self.wrist_motor: TalonFX = TalonFX(
+            config.wrist_id,
+            config.foc_active,
+            inverted = False,
+            config = config.WRIST_CONFIG
+        )
+
+        self.coral_in_wrist: bool = False
+        self.wrist_angle: radians = 0
+        self.target_angle: radians = 0
+        self.wrist_moving: bool = False
+        self.coral_in_feed: bool = False
+
+    def init(self):
+        self.feed_motor.init()
+        self.wrist_motor.init()
+
+#side roller put it in, side roller kick it out after releasing it, have
+
+# feed
+
+    def feed_in(self):
+        self.feed_motor.set_target_velocity(config.wrist_feed_vel)
```

Comment:
Let's put some docstrings here. What does this method do?

Reply:
When does this command end. Does it simply spin the feed motors or does it do it to reach a goal. Put that in the docstring.